fjord/suggest/apps.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from django.apps import AppConfig
+from django.conf import settings
+
+from fjord.suggest import _PROVIDERS
+from fjord.base.plugin_utils import load_providers
+
+
+logger = logging.getLogger('i.suggest')
+
+
+class SuggestConfig(AppConfig):
+    name = 'fjord.suggest'
+
+    def ready(self):
+        # Load providers and store them in _PROVIDERS stomping on
+        # whatever was there.
+        _PROVIDERS[:] = load_providers(settings.SUGGEST_PROVIDERS, logger)
```

Comment:
For the record, the `_PROVIDERS[:]` on the left-hand side means "replace the contents of `_PROVIDERS` with the value on the right hand side". Thus `_PROVIDERS` points to the same list, but we've changed the contents of the list. It's a cheap way of getting around labeling it as a "global" and then replacing it verbatim which has a bunch of unexciting consequences related to importing names and being difficult to mock and such.